common/src/Microsoft.Azure.IIoT.Hub.Processor/src/Services/EventProcessorFactory.cs

```diff
@@ -299,6 +306,11 @@ private class EventProperties : IDictionary<string, string> {
             private readonly long? _interval;
             private readonly Stopwatch _sw;
             private readonly PartitionContext _partitionContext;
+            private static readonly Gauge kEventProcessorDetails = Metrics
+                .CreateGauge("iiot_event_processor_info", "details about event processor",
+                    new GaugeConfiguration {
+                        LabelNames = new[] { "id", "partition", "status" }
```

Comment:
Can you please use `partitionId` instead of `partition` to align this metric key with diagnostics keys that are delivered with logs.

Also please change logs on line `159` and `165` to use `{partitionId}` instead of `{partition}`. This one is my mistake.

Reply:
done